pre_commit_hooks/pretty_format_json.py

```diff
@@ -17,36 +15,38 @@ def pairs_first(pairs):
         if sort_keys:
             after = sorted(after, key=lambda x: x[0])
         return OrderedDict(before + after)
-    return six.text_type(simplejson.dumps(
-        simplejson.loads(
-            contents,
-            object_pairs_hook=pairs_first,
-        ),
+    json_pretty = json.dumps(
+        json.loads(contents, object_pairs_hook=pairs_first),
         indent=indent,
         ensure_ascii=ensure_ascii,
-    )) + "\n"  # dumps does not end with a newline
+    )
+    # Ensure unicode under Py2
+    if sys.version_info.major == 2:
+        json_pretty = json_pretty.decode()
+    json_pretty += '\n'  # dumps do not end with a newline
```

Comment:
the comment was changed here, though I think the lhs has better grammar

Reply:
Yeah intentional :)

Reply:
I guess it makes more sense to me that `dumps` is a singular noun ("dump string")